addons/gitlab/tests/utils.py

```diff
@@ -1,5 +1,8 @@
 import mock
```

Comment:
Wait shouldn't this be `addons/github/tests/utils.py`, not `addons/gitlab/tests/utils.py`?